src/lights/Light.js

```diff
@@ -1,37 +1,36 @@
 import { Object3D } from '../core/Object3D.js';
 import { Color } from '../math/Color.js';
 
-function Light( color, intensity = 1 ) {
+class Light extends Object3D {
 
-	Object3D.call( this );
+	constructor( color, intensity = 1 ) {
 
-	this.type = 'Light';
+		super();
 
-	this.color = new Color( color );
-	this.intensity = intensity;
+		this.type = 'Light';
+		Object.defineProperty( this, 'isLight', { value: true } );
 
-}
-
-Light.prototype = Object.assign( Object.create( Object3D.prototype ), {
+		this.color = new Color( color );
+		this.intensity = intensity;
 
-	constructor: Light,
+		this.receiveShadow = undefined;
```

Comment:
Where did this come from?

Reply:
These are from old code that was there when I originally did the change.
[This commit](https://github.com/mrdoob/three.js/commit/737bf8c072fce3caee909d130ee20e7c2da798fb#diff-2d334ac590baafa2d643e82461ca77be91a3f831d6c79897fa3ccebb39c508c5) was the change.
I can take then out.